decidim-meetings/app/views/decidim/meetings/meetings/_meeting_poll_actions.html.erb

```diff
@@ -0,0 +1,15 @@
+<% extra_classes = local_assigns.fetch(:mobile, false) ? " block md:hidden" : " hidden md:block" %>
+<% if meeting.poll&.has_questions? %>
+  <section class="layout-aside__section layout-aside__buttons<%= extra_classes %>">
+    <%= action_authorized_link_to :reply_poll, meeting_polls_answers_path(meeting), class: "button button__xl button__secondary w-full", data: { "redirect_url" => meeting_polls_answers_path(meeting) }, resource: meeting do %>
+      <span>
+        <%= meeting.poll.has_open_questions? ? t("reply_poll", scope: "decidim.meetings.meetings.meeting") : t("view_poll", scope: "decidim.meetings.meetings.meeting") %>
+      </span>
+    <% end %>
+    <% if !allowed_to?(:reply_poll, :meeting, meeting:) && allowed_to?(:update, :poll, meeting:) %>
```

Comment:
The first `allowed_to?` is necessary?

Reply:
I've added the admin poll button in the meeting show view only if the admin is not allowed to reply the poll because the only other link to administrate the poll is in that reply view. But maybe it makes more sense to always have this link there for admins.

Reply:
Let's leave this as is for the moment. I will want to see this with real world usage before changing it. Having the "Admin poll" button only when the admin can't reply it sounds good for now.